examples/research_projects/onnxruntime/unconditional_image_generation/train_unconditional.py

```diff
@@ -386,13 +386,13 @@ def main(args):
         ]
     )
 
-    def transforms(examples):
+    def transforms_images(examples):
```

Comment:
nit: how do you feel about slightly renaming to `transform_images`, so it's a verb?

```suggestion
    def transform_images(examples):
```